src/app/store.ts

```diff
@@ -1,10 +1,20 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
 import counterReducer from '../features/counter/counterSlice';
+import usersReducer from '../features/users/usersSlice';
+import authorReducer from '../features/author/authorSlice';
+import postsReducer from '../features/posts/postsSlice';
+import selectedPostReucer from '../features/selectedPost/selectedPostSlice';
```

Comment:
There is a typo in the variable name 'selectedPostReucer'. It should likely be 'selectedPostReducer'.

Reply:
fixed